apps/heureka-next/src/components/Services/ServicesList/ServicesList.test.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
```

Comment:
This relates to the last comment about the header... hier is different... Greenhouse vs Juno